src/main/java/com/amazon/opendistroforelasticsearch/sql/plugin/RestSqlAction.java

```diff
@@ -104,7 +104,7 @@ protected RestChannelConsumer prepareRequest(RestRequest request, NodeClient cli
 
             if (request.path().endsWith("/_explain")) {
                 final String jsonExplanation = queryAction.explain().explain();
-                return sendResponse(jsonExplanation, OK);
+                return channel -> channel.sendResponse(new BytesRestResponse(OK, "application/json; charset=UTF-8", jsonExplanation));
```

Comment:
Probably we need to add some tests to catch this kind of a change if it didn't break any of the existing ones.